EventFilter/Utilities/plugins/RecoEventOutputModuleForFU.h

```diff
@@ -68,7 +80,8 @@ namespace evf {
     jsoncollector::DataPointDefinition outJsonDef_;
     unsigned char* outBuf_=nullptr;
     bool readAdler32Check_=false;
-
+    struct flock data_rw_flk;
```

Comment:
Might I suggest that the member variable follow the naming conventions of this class, i.e. `dataRWFlk_` or similar?

Also, `struct` is not needed.